src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -1689,11 +1704,25 @@ namespace HVACUnitarySystem {
 			} else {
 			}}
 
-			// IF small loads to meet, just shut down unit
+			// IF small loads to meet or not converging, just shut down unit
 			if ( std::abs( ZoneLoad ) < Small5WLoad ) {
 				ZoneLoad = 0.0;
 				CoolingLoad = false;
 				HeatingLoad = false;
+			} else if( iterationCounter  > 4 ) { // attempt to lock output (air flow) if oscillations are detected
+				if( QToCoolSetPt < 0.0 ) {
+					HeatingLoad = false;
+					CoolingLoad = true;
+					ZoneLoad = QToCoolSetPt;
+				} else if( QToHeatSetPt > 0.0 ) {
+					HeatingLoad = true;
+					CoolingLoad = false;
+					ZoneLoad = QToHeatSetPt;
+				} else {
+					HeatingLoad = false;
+					CoolingLoad = false;
+					ZoneLoad = 0.0;
+				}
```

Comment:
@rraustad I'm a little uncomfortable with arbitrarily choosing 4 iterations as the time to lock down.  Handing this off to @Myoldmopar and @EnergyArchmage to decide if this should go in 8.5 or if it needs more thought.